airtime_mvc/application/forms/LiveStreamingPreferences.php

```diff
@@ -153,53 +154,7 @@ public function updateVariables()
 
     public function isValid($data)
     {
-        $isValid = parent::isValid($data);
-        $master_source_port = $data['master_source_port'];
-        $show_source_port = $data['show_source_port'];
-
-        if ($master_source_port == $show_source_port && $master_source_port != "") {
-            $element = $this->getElement('show_source_port');
-            $element->addError(_("You cannot use same port as Master DJ port."));
-            $isValid = false;
-        }
-        if ($master_source_port != "") {
-            if (is_numeric($master_source_port)) {
-                if ($master_source_port != Application_Model_StreamSetting::getMasterLiveStreamPort()) {
-                    $sock = socket_create(AF_INET, SOCK_STREAM, SOL_TCP);
-                    try {
-                        socket_bind($sock, 0, $master_source_port);
-                    } catch (Exception $e) {
-                        $element = $this->getElement("master_source_port");
-                        $element->addError(sprintf(_("Port %s is not available"), $master_source_port));
-                        $isValid = false;
-                    }
-
-                    socket_close($sock);
-                }
-            } else {
-                $isValid = false;
-            }
-        }
-        if ($show_source_port != "") {
-            if (is_numeric($show_source_port)) {
-                if ($show_source_port != Application_Model_StreamSetting::getDjLiveStreamPort()) {
-                    $sock = socket_create(AF_INET, SOCK_STREAM, SOL_TCP);
-                    try {
-                        socket_bind($sock, 0, $show_source_port);
-                    } catch (Exception $e) {
-                        $element = $this->getElement("show_source_port");
-                        $element->addError(sprintf(_("Port %s is not available"), $show_source_port));
-                        $isValid = false;
-                    }
-                    socket_close($sock);
-                }
-            } else {
-                $isValid = false;
-            }
-        }
-
-        return $isValid;
-
+        return $isValid = parent::isValid($data);
```

Comment:
This will always return `true` since the assigning the results of `parent::isValid($data)` always evaluates to `true`.

I you want to just have the parents `isValid()` method do it's thing I would recommend removing the whole method fom the subclass.